dashboard/src/components/Accordion/Accordion.tsx

```diff
@@ -126,8 +126,10 @@ const AccordionBuildsTrigger = ({
       <TableCell>
         {triggerInfo.status === 'valid' ? (
           <MdCheck className="text-green" />
-        ) : (
+        ) : triggerInfo.status === 'invalid' ? (
           <MdClose className="text-red" />
+        ) : (
+          <span>-</span>
```

Comment:
Just a nit, to avoid these nested ternaries you can cerate 3 variables to make it more readable

like

const isValid = trigerInfo.status === 'valid'
const isInvalid = trigerInfo.status === 'invalid'
const isOther = !isValid && !isInvalid


and go like

isValid &&
isInvalid &&
isOther &&